audit/audit.go

```diff
@@ -59,6 +82,45 @@ func InfoFromContext(ctx context.Context) *Info {
 	return nil
 }
 
+type auditFormatter struct {
+	underlyingFormatter logrus.Formatter
+}
+
+func (a auditFormatter) Format(entry *logrus.Entry) ([]byte, error) {
+	// Make log level predictable for easier replacement later
+	// (then level color for colored output is known beforehand, which is 36 for info)
+	// See sirupsen/logrus@v1.9.0/text_formatter.go:19
+	entry.Level = logrus.InfoLevel
+
+	formattedEntry, err := a.underlyingFormatter.Format(entry)
+	if err != nil {
+		return nil, err
+	}
+	// Replace log level in formatted log message
+	switch a.underlyingFormatter.(type) {
+	case *logrus.JSONFormatter:
```

Comment:
I would create a `NewAuditFormatter(logrus.StandardLogger().Formatter)` that returns a specific AuditFormatter. In the New func you can do the switch and then each formatter has a simple Format function.